lib/file_management/manage_work_file.py

```diff
@@ -72,16 +72,17 @@ def _move_stu_file(path,filename):
     shutil.copyfile(os.path.join(path,filename),os.path.join(path,"ta","Assignment",dirname,filename))
     return True
 
-def _check_valid_file_name(draft,listfile):
+def _check_invalid_file_name(draft,listfile):
     # Check that filename is folow draft or not
-    validfile = []
+    invalid = []
     for i in listfile[::]:
```

Comment:
I'm not sure that using `[::]` make anything different from not using it. If not, Removing it would be nice

Reply:
Using [::] when we remove or add an element in the list but don't want to change element in for loop.